### PR TITLE
Update FAQ and Sponsor pages

### PR DESCRIPTION
**Pull request by fboxwala**

### CODE REVIEW

**Comment by fboxwala** [comment]
I also added the "status: hidden" which should fix #173.

I forgot to add it when I made the sponsorship page initially -- when I test the site locally the header only has three things in it. I should fix this.

File: content/pages/sponsors.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 Title: Sponsorship
+status: hidden
 
 Occasionally companies will help us run events by providing us with funding, currently we are working
 on the following projects:
 
-### Technical Workshop Series with BlackBerry
+
+## Technical Workshop Series 
+#### Brought to you in partnership with ![BlackBerry](http://csclub.uwaterloo.ca/~fhboxwal/bb_logo.jpg)
```
Comment (meltedesco) [inline content/pages/sponsors.md]: We should be hosting this image ourselves.

I've added the image, so when we next deploy it should be hosted at: http://wics.uwaterloo.ca/images/bb_logo.jpg

**Comment by fboxwala** [comment]
This is the correct thing right

**Comment by meltedesco** [comment]
Looks good to me.

**Pull request merged**